<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerioViewClass</class>
 <widget class="QWidget" name="PerioViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1243</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1243</width>
    <height>1424</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PerioView</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>1196</width>
     <height>801</height>
    </rect>
   </property>
   <widget class="PerioStatusView" name="maxilla"/>
   <widget class="PerioStatusView" name="mandibula"/>
  </widget>
  <widget class="QPushButton" name="upperButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Горна зъбна редица</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="lowerButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>260</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Долна зъбна редица</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="calAvg">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>60</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="PerioTable" name="tableView">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>802</width>
     <height>203</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PerioStatusView</class>
   <extends>QWidget</extends>
   <header>Widgets/periostatusview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PerioTable</class>
   <extends>QTableView</extends>
   <header>Widgets/PerioTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
